.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Can you add `.idea` to the top level `.gitignore` file? That should remove these temporary files.